tools/sr_assembly/velvetg_wrapper.py

```diff
@@ -29,7 +29,7 @@ def __main__():
         proc = subprocess.Popen(args=cmdline, shell=True, stderr=subprocess.PIPE)
         returncode = proc.wait()
         # get stderr, allowing for case where it's very large
-        stderr = ''
+        stderr = b''
```

Comment:
Can you also change `stop_err` to just do `sys.exit("%s\n" % msg)` so we get a non-zero exit code ?